HW_1/task.py

```diff
@@ -0,0 +1,12 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.--------------------------------------------------------------------
+
+num = int(input("Трехзначное число: "))
+res = 0
+if num > 999 or num < 100:
+    print("Не трехзначное число")
+else:
+    while num > 0:
+        a = num % 10
+        res += a
```

Comment:
Это можно объединить
```suggestion
        res += num % 10
```